woocommerce/woocommerce-android
# [Coupons] Product selector search

This PR adds product search to the product selector.

![image](https://user-images.githubusercontent.com/1522856/173236826-845569fc-a304-42ee-b88c-2329fa55a68f.png)

**To test:**
1. Go to Coupons and tap on one
2. Select Edit coupon from the menu
3. Tap on the Edit products button
4. Type a query in the search field
5. Notice the search returns the products matching the query

~**Note:** Before merging, please wait until the [FluxC PR](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2438) is merged first and the hash reference is updated to `trunk`.~
